ModelFiles/xml/yeastGEM.xml

```diff
@@ -258,7 +258,12 @@
           </rdf:RDF>
         </annotation>
       </species>
-      <species metaid="s_0013__91__c__93__" sboTerm="SBO:0000247" id="s_0013__91__c__93__" name="(5S,6S)-di-HETE [cytoplasm]" compartment="c" hasOnlySubstanceUnits="false" boundaryCondition="false" constant="false" fbc:charge="0" fbc:chemicalFormula="C20H32O4">
+      <species metaid="s_0013__91__c__93__" sboTerm="SBO:0000247" id="s_0013__91__c__93__" name="(5S,6S)-di-HETE [cytoplasm]" compartment="c" hasOnlySubstanceUnits="false" boundaryCondition="false" constant="false" fbc:charge="-1" fbc:chemicalFormula="C20H31O4">
+        <notes>
+          <body xmlns="http://www.w3.org/1999/xhtml">
+            <p>NOTES: | metFormula curated (PR #222) | metCharge curated (PR #222)</p>
```

Comment:
The note field here (and in many other cases below) looks odd, as there's no need to start with a `|` character if there was nothing in the note before.

Reply:
@eiden309 This can be solved by including the code that I also suggested for PR  https://github.com/SysBioChalmers/yeast-GEM/pull/220#discussion_r426143440.

Reply:
Noted on this issue, the latest commit with changes to the script should resolve this issue